src/main/java/cloud/orbit/actors/extensions/metrics/dropwizard/OrbitActorExtension.java

```diff
@@ -48,25 +49,26 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.TimeUnit;
 
 /**
- * A pipeline Extension to collect the actor's metrics: number of actor, actor lifespans distribution, actor message received rate
+ * A pipeline Extension to collect the actor's metrics: number of actor, actor lifespans distribution, actor message
+ * received rate
  *
  * Created by jgong on 12/16/16.
  */
 public class OrbitActorExtension extends NamedPipelineExtension implements LifetimeExtension
 {
     private static final Logger logger = LoggerFactory.getLogger(OrbitActorExtension.class);
-  
+
     public static final String ACTOR_METRICS_PIPELINE_NAME = "actor-metrics-pipeline";
 
     //Actor Counter, per actor type and node, actor count breakdown to each type
     Map<String, Counter> actorCounters = new HashMap<>();
 
-    //Timer, per actor instance
-    Map<String, Long> actorStartTimes = new ConcurrentHashMap<>();
-
-    Map<String, Histogram> actorLifeSpanHistograms = new HashMap<>();
+    //Timer context, per actor instance
+    Map<String, Long> actorLifespanStartTimes = new ConcurrentHashMap<>();
```

Comment:
1. you can do Timer.Context.time()/stop(), then we have to keep the reference for the contexts plus it only does timer in nano-seconds
2. changed the variable name and added some comments